sdk/lib/drivers/sound/mmixer/controls.c

```diff
@@ -217,12 +222,11 @@ MMixerAddMixerControl(
                     return MM_STATUS_NO_MEMORY;
                 }
 
-                Value = Range->Bounds.SignedMinimum;
+                Value = Range->Bounds.UnsignedMinimum;
```

Comment:
Note that `Value` is declared as (signed) `LONG`, see line 186.
Is this the change that allows the removal of the "- 1" below?

Reply:
>Note that Value is declared as (signed) LONG, see line 186.

Indeed. Then it can be changed to `ULONG`, because it contains unsigned values anyway.

>Is this the change that allows the removal of the "- 1" below?

No, the change which replaces that hack (fixes the problem properly) is 
<img width="518" height="66" alt="изображение" src="https://github.com/user-attachments/assets/d5bc56d4-10d0-47bc-bb1b-283f69da33ee" />

Reply:
@oleg-dubinskiy did you mean to change Line 186 from:
```LONG Value;```
to
```ULONG Value;``` ?